src/components/DatoVelger.tsx

```diff
@@ -23,11 +25,11 @@ const DatoVelger = (props: DatoVelgerProps) => {
 
   return (
     <div className={'skjemaelement'}>
-      <Label htmlFor="datoId">{props.label}</Label>
+      <Label htmlFor={props.id ?? 'datoId'}>{props.label}</Label>
```

Comment:
Synes den burde defaulte til datoVelgerId for å potensielt krangle med andre komponenter